frontend/src/locales/en.json

```diff
@@ -120,6 +120,7 @@
     "busy": "Busy",
     "caldav": "CalDAV",
     "calendar": "Calendar",
+    "calendars": "Calendars",
```

Comment:
Should we rather use i18n plural functionality here?

Reply:
Good catch!